### ISSUE TITLE
No quick selection button on new site?

### ISSUE THREAD

**Issue by theLMGN** [google/fonts#2384]
Before, you used to be able to click the little pink + icon to select a font quickly, and add it to your list
![image](https://user-images.githubusercontent.com/12257112/76106883-a76c8f00-5fcf-11ea-8a55-c7e3c17cb0e1.png)
Now, there is no button and you have to click on the font to go to its page, select the styles you want, press the browsers back button, and then you can continue.
![image](https://user-images.githubusercontent.com/12257112/76106918-bbb08c00-5fcf-11ea-8024-81edb5dcf980.png)

I also find it annoying that the button in the top left of the font page used to be select this font, now it downloads a ZIP file.
![image](https://user-images.githubusercontent.com/12257112/76107024-ed295780-5fcf-11ea-95ce-c5ea34bbd039.png)
![image](https://user-images.githubusercontent.com/12257112/76107032-ee5a8480-5fcf-11ea-808c-53d984ee90ea.png)

**Comment by MentalGear** [comment]
Yes, this absolutely broke the service for me. Not using it anymore.

**Comment by tvjudge** [comment]
yes this definately sux extremely non-intuitive and frustrating

**Comment by tvjudge** [comment]
yes this definately sux extremely non-intuitive and frustrating
